packages/design-system/src/components/Table/TableBody.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TableBody = ({ children, className, ...attributeOptions }) => {
```

Comment:
Is `attributeOptions` just any additional props? Usually I've seen this named `other` or named the component like `tableBodyProps`

Reply:
`attributeOptions` refers to any other attributes that get added to TableBody which have be picked up by the destructuring assignment from the prop objects.

Reply:
Right I understand that, I'm asking what values does `attributeOptions` hold right now? `other` is the common name for destructured variables that are usually empty (except when users pass on additional props), and a naming pattern like `tableBodyProps` would be specific to a destructured variable that holds a group of props. I think this naming could be more helpful.

Reply:
@nichia I dont think this question was answered, what values is in `attributeOptions`. I ask because depending on that, I think the variable could be renamed to `other` or something more specific

Reply:
https://github.com/CMSgov/design-system/pull/706#discussion_r440242715
That is a good example of why it might be better to explicitly define certain variables and make the destructured variable naming very clear.

Reply:
Updated to have more explicit usage of the variables.